src/components/App/App.js

```diff
@@ -22,15 +25,21 @@ function App() {
     e.preventDefault();
```

Comment:
e.preventDefault();?is it possible to change button type?

Reply:
Added type="text" to button, but it did not help. Left as it was with e.preventDefault().